index.js

```diff
@@ -1322,6 +1328,9 @@ var BP = Buffer.prototype
 
 /**
  * Augment a Uint8Array *instance* (not the Uint8Array class!) with Buffer methods
+ *
+ * This function is also used externally by `typedarray-to-buffer`.
+ *
  */
 Buffer._augment = function _augment (arr) {
   arr.constructor = Buffer
```

Comment:
`typedarray-to-buffer` is switched over now, so we don't need this line anymore.